### ISSUE TITLE
Unable to get trophies list for a username

### ISSUE THREAD

**Issue by mohsiniscoding** [achievements-app/psn-api#22]
Here is the code I'm using, when I use "me" as accountID, I get result:
`{ trophyTitles: [], totalItemCount: 0 }`

when I use other username, it does not work

```
const psn = require("psn-api");

async function start() {

    // This is the value you copied from the previous step.
    const myNpsso = "TOKEN HERE";

    // We'll exchange your NPSSO for a special access code.
    const accessCode = await psn.exchangeNpssoForCode(myNpsso);
    // 🚀 We can use the access code to get your access token and refresh token.
    const authorization = await psn.exchangeCodeForAccessToken(accessCode);

    // This returns a list of all the games you've earned trophies for.
    const trophyTitlesResponse = await psn.getUserTitles(
        { accessToken: authorization.accessToken },
        "ViensDanser"
    );

    console.log(trophyTitlesResponse);

}

start();
```

Output:
logging
```
{
  error: {
    referenceId: 'f842a5e4-38c6-11ec-97f6-b71e9c99aa8c',
    code: 2240513,
    message: 'Bad Request (path: sort)'
  }
}
```

**Comment by wescopeland** [comment]
I've figured this out.

This is the endpoint to hit:
POST https://m.np.playstation.net/api/search/v1/universalSearch

Send this JSON payload:
```json
{
  "searchTerm": "ViensDanser",
  "domainRequests": [
    {
      "domain": "SocialAllAccounts"
    }
  ]
}
```

In the response you will see:

```json
"accountId": "7867532288966013754"
```

I will be working on an update to the library to add an easy function to call to do this automatically.

**Comment by wescopeland** [comment]
This has landed in version 2.2.0 🎉

The new function is `makeUniversalSearch`. Here is a usage example:

```ts
import { makeUniversalSearch } from "psn-api";
const response = await makeUniversalSearch(
  authorization,
  "ViensDanser",
  "SocialAllAccounts"
);
```

Docs can be found here: https://psn-api.achievements.app/api-docs/universal-search#makeuniversalsearch
